assets/mappings/bi-banks.json

```diff
@@ -0,0 +1,102 @@
+{
+  "3da1697b-59b0-5bb4-8ec8-08125c3390ce": "americanexpress",
+  "c46cf6d6-2d7d-5ba6-9f78-e0bcb3597921": "arkea",
+  "7fc67931-e08f-5d13-b870-02cb13269c0d": "axabanque102",
+  "4878d90f-a3d5-5dc9-9c7a-c822484890b8": "banquecasino",
+  "de508df8-aa37-537e-a05f-1379526dfa84": "banquepopulaire",
+  "6d1876e3-06ea-5d96-aaed-c53f51aacc2b": "barclays136",
+  "646e8e7f-0163-5d75-a95a-27438f82c1de": "bforbank97",
+  "f711dd7a-6289-5bda-b3a4-f2febda8c046": "bnpparibas82",
+  "4aa3d2e6-1ec6-59c5-9e4d-8b28ecdd31cf": "bolden",
+  "07d76adf-ae35-5b38-aca8-67aafba13169": "boursorama83",
+  "86504cdf-cda6-54e5-b51a-a96e17cc8410": "bred",
+  "13949b61-e3e6-50ed-9d82-451943ec35d0": "caissedepargne1",
+  "3000b0e0-8a93-59e7-a351-ad5625ec19a3": "carrefour159",
+  "0d70c378-37b7-5e1f-af39-0d20eea24b19": "cetelem",
+  "e9606d38-6b0f-5f76-b573-61a4d00a927d": "cic63",
+  "025daf3c-0140-5f6d-a4a8-b7423c97f745": "comptenickel168",
+  "d840908d-5157-5819-8296-474fa4534564": "caatlantica3",
+  "bd6e3615-a45f-5f9e-80c2-a6714081d9ce": "creditcooperatif148",
+  "cdff375f-8353-56dc-9d58-1c159f21740e": "cdngroup88",
+  "cc9da113-55dc-50bd-bf9d-9261dfdfbb31": "creditmaritime",
+  "fc6e059f-c3e6-52d7-8139-6e5e0297fe24": "cic45",
+  "ab3acf4c-cfbc-5b9f-b6d9-cd4eb1838cd4": "cic45",
+  "f1309fa1-f1fd-51fd-ad9a-a7aba985ce67": "cic45",
+  "b247dd6e-4ccc-598c-9f12-ea740465e2f0": "fortuneo84",
+  "5b2ebdf0-9ab2-5e0d-a6d4-26c255f9d2d0": "hellobank145",
+  "9efdf972-573a-50e0-9245-2d9e310d7f61": "hsbc119",
+  "a3959e20-9fc8-53e8-a442-66e4940db567": "ingdirect95",
+  "33f88657-7ab0-5dbe-b8c3-c98e614afc28": "labanquepostale44",
+  "ad5928c7-89ce-5977-a66d-3360e3aa4029": "lcl-linxo",
+  "cd1f4851-ddfe-535b-9c88-e50dbc59d864": "maifviebank",
+  "306b84f8-78ad-5a90-93ed-3e5cdb01344a": "monabanq96",
+  "6d4f9fba-7897-52e3-aec0-29fdbabd5555": "cdngroup109",
+  "f5c29767-1bc8-5337-9e4e-68a0fbd91c9a": "societegenerale",
+  "338178e6-3d01-564f-9a7b-52ca442459bf": "bankingconnectortestwebauth",
```

Comment:
There was a mistake in the file we gave you : we have to replace `bankingconnectortestwebauth` with `bankingconnectortest`